packages/frontend/core/src/blocksuite/presets/ai/chat-panel/index.ts

```diff
@@ -122,6 +122,7 @@ export class ChatPanel extends WithDisposable(ShadowlessElement) {
         this._chatSessionId = history.sessionId;
         this.chatContextValue.chatSessionId = history.sessionId;
         items.push(...history.messages);
+        AIProvider.LAST_ROOT_SESSION_ID = history.sessionId;
```

Comment:
why?

Reply:
Each doc will have one and only one root chat session, which is the session corresponding to the conversation in the right sidebar. It is needed when conducting conversations or forking sessions. This session id needs to be recorded when loading the sidebar for the first time or switching doc.